src/Compilers/CSharp/Portable/Syntax/StackAllocArrayCreationExpressionSyntax.cs

```diff
@@ -13,7 +13,7 @@ public StackAllocArrayCreationExpressionSyntax Update(SyntaxToken stackAllocKeyw
     public partial class LocalDeclarationStatementSyntax
```

Comment:
>LocalDeclarationStatementSyntax [](start = 25, length = 31)

Not caused by this PR: I didn't understand why this type is in this file (for stackalloc). Seems like it should be split into a separate file (for local declaration).